modules/virt-virtctl-commands.adoc

```diff
@@ -73,6 +73,15 @@ You can use `virtctl create` commands to create manifests for virtual machines,
 |`virtctl create vm --name <vm_name>`
 |Create a VM manifest, specifying a name for the VM.
 
+|`virtctl create vm --user <user_name> --ssh-key\|password-file=<value>`
+|Create a VM manifest with a cloud-init configuration to create the selected user and either add an SSH public key from the supplied string, or a password from a file.
+
+|`virtctl create vm --access-cred type:<type>,src:<secret>,user:<user_name>`
```

Comment:
`user:<user_name>` is not supported for `type:password`, since the referred to secret contains a mapping of users to passwords. You have to split these examples.

Reply:
I did not realize that, I split the examples as you suggested, please take a look at the updated version.